bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -152,7 +152,7 @@ public function treeGetChildsByIdAction(Request $request, EventDispatcherInterfa
             if ($request->get('view')) {
                 $cv = Element\Service::getCustomViewById($request->get('view'));
 
-                if ($cv['classes']) {
+                if (isset($cv['classes']) && is_array($cv['classes'])) {
```

Comment:
```suggestion
                if (!empty($cv['classes'])) {
```
otherwise an empty array would now only filter folders

Reply:
puah. that's a good catch! thank you.

Reply:
Three commits to fix this. Heavy duty :D